src/notation/view/pianokeyboard/pianokeyboardpanelcontextmenumodel.cpp

```diff
@@ -167,6 +203,31 @@ MenuItem* PianoKeyboardPanelContextMenuModel::makeNumberOfKeysItem(const muse::T
     return item;
 }
 
+MenuItem* PianoKeyboardPanelContextMenuModel::makeToggleNotatedPitchItem(const muse::TranslatableString& title, bool isNotatedPitch)
+{
+    UiAction action;
+    action.title = title;
+    action.code = SET_NOTATED_PITCH_CODE;
+    action.checkable = Checkable::Yes;
```

Comment:
Perhaps, instead of "checkable", we should use "selectable". That is not a property of UiAction but of MenuItem. 

So: `item->setSelectable(true)` to mark the item as selectable, and then `bool selected = …; item->setSelected(selected)` to set the state.